src/Grok.Net.Tests/UnitTests.cs

```diff
@@ -304,6 +305,35 @@ public void Parse_Base64String_Patterns(string base64String)
             Assert.Single(grokResult);
         }
 
+        [Fact]
+        public void Parse_Multiline()
+        {
+            // Arrange
+            const string timeKeyword = "loggingTime";
+            const string messageKeyword = "message";
+            const RegexOptions options = RegexOptions.Singleline;
```

Comment:
Is it ok that the test is about Multiline but here we got Singleline option?

Reply:
Yeah, I'll change the test name. Essentially, we're testing that grok will be generated as if newlines don't exist in the string. So, it's a multiline string, but is using a singleline mode. Is name `Parses_Multiline_as_a_Single_Line` OK?